doc/api/index.rst

```diff
@@ -17,35 +17,53 @@ All plotting is handled through the :class:`pygmt.Figure` class and its methods.
 
     Figure
 
-Plotting data and laying out the map:
+Plotting tabular data:
 
 .. autosummary::
     :toctree: generated
 
-    Figure.basemap
-    Figure.coast
-    Figure.colorbar
     Figure.contour
+    Figure.histogram
+    Figure.meca
+    Figure.plot
+    Figure.plot3d
+    Figure.rose
+    Figure.velo
+    Figure.wiggle
+
+Plotting gridded data:
```

Comment:
Just as an throwaway idea, would 'raster' be a better term than 'gridded'? I know the 'Data Processing section uses 'grid', but the GIS person in me is thinking about raster/vector instead of gridded/tabular :slightly_smiling_face:

```suggestion
Plotting raster data:
```